.travis.yml

```diff
@@ -26,7 +26,11 @@ before_cache:
   - docker exec --user=postgres postgresql pg_dump rapidpro > $HOME/postgresql/rapidpro.sql
 
 before_script:
-  - LATEST_TAG=$(curl -s https://api.github.com/repos/$REPO/tags  | jq -r '.[0].name')
+  - echo "https://api.github.com/repos/$REPO/tags"
+  - jq --version
+  - curl -s https://api.github.com/repos/$REPO/tags | tee ./tags.json
+  - head ./tags.json
+  - LATEST_TAG=$(cat ./tags.json | jq -r '.[0].name')
```

Comment:
Travis is hitting a fair amount of rate limiting issues for its IPs. I'm leaving this here because this way it's easier to spot when a build fails because the request to get the latest tag is failing due to rate limiting.